### DISCUSSION TITLE
Pageup and pagedown to scroll quicklook panel

### DISCUSSION THREAD

**Discussion by M0rtenB** [mikekazakov/nimble-commander#221] [Ideas]
When I use the quicklook/preview functionality in the opposite panel, it would be nice if pageup and pagedown were temporarily  remapped to scroll the preview window, i.e. to navigate the pages of a multipage pdf. This is similar to how it works with quicklook in Finder and Alfred.

**Comment by mikekazakov** [comment]
Please take a look at F.A.Q.: https://github.com/mikekazakov/nimble-commander/discussions/136.  
I'm not aware of a way to implement it, but any suggestions or tips would be welcome.
